packages/cli/lib/projects.js

```diff
@@ -268,6 +274,13 @@ const pollBuildStatus = async (accountId, name, buildId) => {
             }
           });
         }
+
+        if (!autoDeployEnabled) {
+          resolve(buildStatus);
+        }
+      }
+
+      if (autoDeployEnabled && deployStatusTaskLocator) {
```

Comment:
What happens if `deployStatusTaskLocator` does not exist? Should we treat that as a BE internal error?

Reply:
Yeah, I'll check for a valid `deployStatusTaskLocator`. Working on the messaging/error state part of this now